src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -336,6 +378,20 @@ export class HostsComponent extends ListWithDetails implements OnInit {
     });
   }
 
+  transformHostsData() {
+    if (this.orchStatus?.available) {
+      _.forEach(this.hosts, (hostKey) => {
+        hostKey['memory_total_bytes'] = hostKey['memory_total_kb'] * 125;
+        hostKey['raw_capacity'] = hostKey['hdd_capacity_bytes'] + hostKey['flash_capacity_bytes'];
+      });
+    } else {
+      for (let column = 5; column < this.columns.length; column++) {
+        this.columns[column]['prop'] = '';
+        this.columns[column]['cellTemplate'] = this.orchTmpl;
+      }
```

Comment:
What's this for?

Reply:
It's for  showing `Unavailable` for gather facts columns when orch is not available.